doc/changelog.rst

```diff
@@ -134,8 +134,13 @@ decodes datetime as naive by default.
 - The ``hint`` option is now required when using ``min`` or ``max`` queries
   with :meth:`~pymongo.collection.Collection.find`.
 - ``name`` is now a required argument for the :class:`pymongo.driver_info.DriverInfo` class.
+- :class:`~pymongo.collection.Collection` and :class:`~pymongo.database.Database`
+now raises an error upon evaluating as a Boolean, please use the
+syntax ``if collection is not None:`` or ``if database is not None:`` as
+opposed to
+the previous syntax which was simply ``if collection:`` or ``if database:``.
+You must now explicitly compare with None.
```

Comment:
These lines won't render properly because of the indentation.

Reply:
Done.